mta/widgetastic/__init__.py

```diff
@@ -37,7 +39,10 @@ class DropdownMenu(Dropdown):
     BUTTON_LOCATOR = (
         ".//span[@class='filter-by']/parent::button | "
         ".//span[@id='sort-by']/parent::button | "
-        ".//button[contains(@class, 'pf-c-context-selector__toggle')]"
+        ".//button[contains(@class, 'pf-c-context-selector__toggle')] | "
+        ".//span[@class='pf-c-select__toggle-icon']/parent::div/parent::button | "
+        ".//span[@class='pf-c-select__toggle-arrow']/parent::button["
+        "contains(@aria-labelledby, 'Filter by Source')]"
```

Comment:
Since we are making change here we need to run all tests and see if all tests are working after this change .